tests/Unit/Plugins/LegacyPluginsTest.php

```
@@ -1,395 +1,361 @@
+});
 
-    protected function tearDown(): void
-    {
-        $this->cleanupTestPlugins();
-        $this->restoreExistingPlugins();
+function vitoPestUnitPluginsLegacyPluginsTestMoveExistingPlugins(): void
+{
+    $pluginsPath = storage_path('plugins');
 
-        parent::tearDown();
+    if (File::exists($pluginsPath)) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\File' in method 'vitoPestUnitPluginsLegacyPluginsTestMoveExistingPlugins'. (undefined)

(StaticAccess)